module/user-profile/helper.js

```diff
@@ -5,188 +5,648 @@
  * Description : All user profile helper related information.
  */
 
-let userManagementService = 
-require(ROOT_PATH +"/generics/services/user-management");
-let entitiesHelper = require(ROOT_PATH+"/module/entities/helper");
+let entitiesHelper = require(ROOT_PATH + "/module/entities/helper");
 
 module.exports = class UserProfileHelper {
 
+     /**
+      * List of user profile.
+      * @method
+      * @name list
+      * @param {Object} [queryParameter = "all"] - Filtered query data.
+      * @param {Object} [projection = {}] - Projected data.   
+      * @returns {Object} returns a entity types list from the filtered data.
+     */
+
+    static list(queryParameter = "all", projection = {}) {
+        return new Promise(async (resolve, reject) => {
+            try {
+
+                if( queryParameter === "all" ) {
+                    queryParameter = {};
+                };
+
+                let userProfileData = 
+                await database.models.userProfile.find(queryParameter, projection).lean();
+
+                return resolve(userProfileData);
+
+            } catch (error) {
+                return reject(error);
+            }
+        })
+
+    }
+
     /**
-   * Create user profile.
-   * @method
-   * @name create
-   * @param  {requestedData}  - requested body.
-   * @returns {json} Response consists of created user profile data.
-   */
+    * Get user profile form data.
+    * @method
+    * @name getForm
+    * @returns {json} Response consists of user form data.
+    */
 
-    static create( requestedData,token ) {
+    static getForm(loggedInUser, appName = "", device = "") {
         return new Promise(async (resolve, reject) => {
             try {
 
-                let userProfileCreationData = 
-                await userManagementService.createUserProfile(
-                    requestedData,
-                    token
-                )
+                let formData =
```

Comment:
@aman-tunerlabs - Can we not have form helpers? The rule is no other module/model query to be done directly in another helper.